package.json

```diff
@@ -13,7 +13,7 @@
     "@blinkmobile/json-as-files": "1.1.1",
     "appdirectory": "0.1.0",
     "are-we-there-yet": "1.1.2",
-    "async": "1.5.2",
+    "async": "2.0.1",
```

Comment:
Nope. From my own testing and the automated tests (which have 100% coverage as far as the way we use async is concerned), there doesn't seem to be any breakage.

We only use a few of async's functions here. 2.x improved the modularity of the project, so there might be some future performance gains for just importing the bits we use, instead of the whole thing. Maybe?